packages/contractkit/test/driver.test.ts

```diff
@@ -0,0 +1,59 @@
+import { Address } from '../src/base'
+import { newKit } from '../src/kit'
+
+test('Test driver', async () => {
+  // Integration
+  // const kit = newKit('http://35.247.98.50:8545')
+  const kit = newKit('http://35.247.74.27:8545')
```

Comment:
What is this? How do you intend on testing this going forward?

Reply:
the file is just an example, not a test...

For testing, use ganache and start from a snapshot that already has all contracts

Reply:
See https://github.com/celo-org/celo-monorepo/issues/472

Reply:
Should we remove this file then?